src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -59,22 +59,26 @@ internal class EslintBridgeClient : IEslintBridgeClient
         private readonly IEslintBridgeProcess eslintBridgeProcess;
         private readonly IEslintBridgeHttpWrapper httpWrapper;
         private readonly IAnalysisConfiguration analysisConfiguration;
+        private readonly IEslintBridgeKeepAlive keepAlive;
         private bool isDisposed;
 
         public EslintBridgeClient(string analyzeEndpoint, IEslintBridgeProcess eslintBridgeProcess, ILogger logger)
-            : this(analyzeEndpoint, eslintBridgeProcess, new EslintBridgeHttpWrapper(logger), new AnalysisConfiguration())
+            : this(analyzeEndpoint, eslintBridgeProcess, new EslintBridgeHttpWrapper(logger), new AnalysisConfiguration(),
+                  new EslintBridgeKeepAlive(eslintBridgeProcess, logger))
```

Comment:
The client is just responsible for creating and disposing the keep-alive handler.

In an earlier version, the keep-alive code was all in this class. This version is simpler. See #3203 for the earlier version.

I considered adding the functionality to the `EslintBridgeProcess` class, but it is already pretty complicated.